<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora PyLine</string>
  </property>
  <widget class="QWidget" name="centralwidget">

    <widget class="QPushButton" name="bt5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>

   <widget class="QPushButton" name="bt6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>

   <widget class="QPushButton" name="bt8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btpunto">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>

   <widget class="QPushButton" name="bt9">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>

   <widget class="QPushButton" name="bt0">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btdiv">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btcal">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btparI">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>

   <widget class="QPushButton" name="bt3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btparD">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btclear">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>

   <widget class="QPushButton" name="bt7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>

   <widget class="QPushButton" name="bt1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btmenos">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btporc">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btpot">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>x²</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btpor">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btpi">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>π</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btcos">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btlog">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btdes">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btrad">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>√</string>
    </property>
   </widget>

   <widget class="QPushButton" name="bt4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btmas">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>

   <widget class="QPushButton" name="bt2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>31</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>

   <widget class="QPlainTextEdit" name="textB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <zorder>btpi</zorder>
   <zorder>btrad</zorder>
   <zorder>bt5</zorder>
   <zorder>bt6</zorder>
   <zorder>bt8</zorder>
   <zorder>btpunto</zorder>
   <zorder>bt9</zorder>
   <zorder>bt0</zorder>
   <zorder>btdiv</zorder>
   <zorder>btcal</zorder>
   <zorder>btparI</zorder>
   <zorder>bt3</zorder>
   <zorder>btparD</zorder>
   <zorder>btclear</zorder>
   <zorder>bt7</zorder>
   <zorder>bt1</zorder>
   <zorder>btmenos</zorder>
   <zorder>btporc</zorder>
   <zorder>btpot</zorder>
   <zorder>btpor</zorder>
   <zorder>btdes</zorder>
   <zorder>bt4</zorder>
   <zorder>btmas</zorder>
   <zorder>bt2</zorder>
   <zorder>textB</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>294</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
